configs/run2_Z_2017_miniIso.json

```diff
@@ -0,0 +1,56 @@
+{
+    "efficiencies": [
+        ["TightIDandMiniIso", "TightID"],
```

Comment:
One last change, since you removed TightIDandMiniIso these would now be MiniIsolationTight for the numerator (similarly for Loose).